templates/spec/controllers/checkout_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'solidus_starter_frontend_helper'
```

Comment:
I kept `solidus_starter_frontend_helper` because it provides it adds our opinionated RSpec configuration to the app. It might be better to keep that configuration isolated from any RSpec configurations that the app's developer adds to the test suite. Let me know if you think that it makes more sense to merge this helper's configuration to the app's `rails_helper` config.